### DISCUSSION TITLE
[Q] What is Linear(30) that is shown here: https://dotnet.stockindicators.dev

### DISCUSSION THREAD

**Discussion by dehghani-mehdi** [DaveSkender/Stock.Indicators#1285] [Help and support]
Is that trendline? I could not find any related info in the documentation.

**Comment by dehghani-mehdi** [comment]
I got it, it's Linear Regression

**Comment by DaveSkender** [comment]
Yah, it's just an [ordinary least-squares linear regression](https://dotnet.stockindicators.dev/indicators/Slope/#content), which is a generated alongside related Slope calculations, and would look like this when plotted.

![](https://github.com/user-attachments/assets/08601a25-d96e-428c-a90b-33602ba48e61)

You can visualize most indicators here: [charts.stockindicators.dev](https://charts.stockindicators.dev)
